packages/xstate-react/src/createActorContext.tsx

```diff
@@ -15,24 +15,31 @@ export function createActorContext<TMachine extends AnyStateMachine>(
     selector: (snapshot: EmittedFrom<TMachine>) => T,
     compare?: (a: T, b: T) => boolean
   ) => T;
-  useActorRef: () => ActorRefFrom<TMachine>;
+  useContext: () => ActorRefFrom<TMachine>;
   Provider: (props: {
     children: React.ReactNode;
+    value?: TMachine | ActorRefFrom<TMachine>;
```

Comment:
This type is incorrect and it was only allowed because we still don't use `strictFunctionTypes: true`.

This `value` is passed to the `useInterpret` and that doesn't accept actor refs - it only accepts machines (and machine factories):

It should be:
```suggestion
    value?: TMachine | (() => TMachine);
```